cli/tests/activate.bats

```diff
@@ -3190,3 +3190,22 @@ EOF
   run "$FLOX_BIN" activate -- printenv FLOX_DISABLE_METRICS
   assert_output "true"
 }
+
+# ---------------------------------------------------------------------------- #
+
+@test "can use fallback interpreter" {
+  project_setup
+  run "$FLOX_BIN" activate --use-fallback-interpreter -- true
+  assert_success
+}
+
+@test "fallback flag activates with rendered interpreter" {
+  project_setup
+
+  # Attempting to use the interpreter bundled with the CLI will fail because
+  # we're overriding the variable for the bundled interpreter store path, so
+  # we should only be able to activate if the fallback interpreter is used.
+  FLOX_INTERPRETER="/foo" run "$FLOX_BIN" activate --use-fallback-interpreter -- true
```

Comment:
nonblocking: is it worth adding
```
  FLOX_INTERPRETER="/foo" run "$FLOX_BIN" activate -- true
  assert_failure
```
to make sure we're not getting false positives for this test?

Reply:
Fixed